### PR TITLE
Created unit tests for all server routes

### PR DESCRIPTION
**Pull request by pachown**

### CODE REVIEW

**Comment by pachown** [comment]
[Server Tests Complete](https://trello.com/c/AmNvN8eu/62-server-tests-complete)

**Review by sabateklu — APPROVED** [review]
great route tests Thomas. approved!

**Comment by pachown** [comment]
[Unit Testing Complete - Thomas](https://trello.com/c/KUm8LSWR/68-unit-testing-complete-thomas)

File: Client/src/components/App.jsx [hunk 1]
```diff
@@ -23,10 +23,9 @@ class App extends React.Component {
   }
 
   getLocation() {
-    return Axios.get('./api/location')
+    return Axios.get('http://localhost:3000/api/location')
       .then((result) => {
```
Comment (this-Cunningham) [inline Client/src/components/App.jsx:28]: can you just do 'axios.get('api/location') ?

File: Client/src/components/App.jsx [hunk 2]
```diff
@@ -47,7 +45,7 @@ class App extends React.Component {
   }
```
Comment (this-Cunningham) [inline Client/src/components/App.jsx:48]: Change to 'api/restaurant'.  I think the ./ is a file path whereas api/restaurant is a route

**Review by this-Cunningham — APPROVED** [review]
Looks good, see if you can figure out a way to route without 'localhost' i think this will be important when deploying.

**Pull request merged**